mono-snapshot

```diff
@@ -4,7 +4,8 @@
 # Copyright 2012 Jo Shields
 # Licensed under WTFPL v2
 
-if [ "$(expr \"$0\" : \".*sh\")" -eq "0" ]
+shtest=$(expr "$0" : ".*sh")
+if [ "$shtest" -eq "0" ]
```

Comment:
This breaks in bash on Ubuntu 14.04 for me.
Changing it to the following solved it:

```diff
-shtest=$(expr "$0" : ".*sh") 
+shtest=$(expr \"$0\" : \".*sh\") 
```

Can you see if this works for ZSH as well?